<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1430</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>615</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1430</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
	background-color: rgb(53, 132, 228)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: #99C1F1
}</string>
   </property>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_1{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_2{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_3{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_4{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_5{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_6{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_7{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_8 {
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_9{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>211</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
	 background-color: #cfcfcf;
     border-radius: 8px;
     font: bold 14px;
     padding: 6px;
	 font-size: 20px;
	border: 1px solid #808080
 }

 QPushButton#pushButton_eq {
     background-color: #ffa000;
 }</string>
    </property>
    <property name="text">
     <string notr="true">=</string>
    </property>
    <property name="autoRepeatInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_plus{
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_minus{
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_mult{
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_div {
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_pow {
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_mod{
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_cos{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_sin{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_tan{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_sqrt{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_ln{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_atan{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_asin{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_acos{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_log{
	background-color: #8B008B;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(255, 255, 255);
	qproperty-alignment: 'AlignRight | AlignVCenter'
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left_bracket">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_left_bracket{
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right_bracket">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>80</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_right_bracket{
	background-color: #FF8C00;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_truncate">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_truncate{
	background-color:#DC143C;
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Delet</string>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_dot{
	background-color: #00FA9A;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>110</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_0{
	background-color: #ADD8E6;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>775</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>20</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>520</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	background : rgb(255, 255, 255);
}</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>111</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_x{
	background-color: #00FA9A;
	border-radius: 8px;
	border: 1px solid #808080
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Область определения X от:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Область определения X до:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>560</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	background : rgb(255, 255, 255);
}
</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionBankCalc">
   <property name="text">
    <string>CreditCalc</string>
   </property>
  </action>
  <action name="actionDepositCalc">
   <property name="text">
    <string>DepositCalc</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
